### ISSUE TITLE
Auction call periods

### ISSUE THREAD

**Issue by ashleyvega** [vegaprotocol/specs#201]
In GitLab by @tamlyn10 on Jan 21, 2020, 10:33

Explore alternative to set end times - 

3 options:
* Indefinite
* Known
* Unpredictable but within a range

**Comment by ashleyvega** [comment]
In GitLab by @tamlyn10 on Jan 21, 2020, 10:34

changed title from **Auction {-randomised -}call periods** to **Auction call periods**

**Comment by ashleyvega** [comment]
In GitLab by @tamlyn10 on Jan 21, 2020, 10:34

changed the description

**Comment by edd** [comment]
Ref #256

**Comment by barnabee** [comment]
I'm not sure this should be closed

@tamlyn10 I think we still want an analysis ticket for [randomised] call periods?

**Comment by tamlyn10** [comment]
yeah, we would want to talk about this still.. who's messing with my analysis tickets?!

**Comment by edd** [comment]
Me

**Comment by edd** [comment]
https://vegaprotocol.slack.com/archives/CAHA5EX0F/p1588864323319400

> I have my doubts as to its it’s usefulness and think it’s clogging up a shared board with people’s good intentions. So far I’ve been linking them to places where the work has advanced, and not closing any that seem to have useful things on

> I think it’s a dumping ground for ideas we’ve considered the existence of but not addressed. It’s like a write only store. I know there are thoughts in there that haven’t been captured elsewhere, I also don’t believe that anyone is reviewing those and putting them in to action. They still exist in the closed state just fine.

**Comment by barnabee** [comment]
Reopened for analysis around gameability of known auction end times and other considerations for call periods. We should know about this in order to complete price monitoring and it's likely necessary to deploy good frequent batch auction markets too.
